Sources/TensorFlow/Core/DifferentialOperators.swift

```diff
@@ -67,8 +71,10 @@ public func valueWithGradient<T, R>(
 ) -> (value: Tensor<R>, gradient: T.TangentVector)
 where T: Differentiable, R: TensorFlowFloatingPoint {
     let (y, pullback) = valueWithPullback(at: x, in: f)
-    precondition(y.rank == 0,
-        "The forward pass computed a non-scalar tensor (shape: \(y.shape)).")
+        precondition(y.rank == 0, """
```

Comment:
I believe that this change and then in line 89 are both indented one level more than they should be.

Reply:
D'oh, yeah. Good catch. Fixed.